addons/custom-zoom/userscript.js

```diff
@@ -18,6 +20,9 @@ export default async function ({ addon, console }) {
   customZoomAreaElement.addEventListener("mousedown", (e) => {
```

Comment:
This is not necessary on the new Blockly, right?

Reply:
Indeed. I'll add a comment so that we don't forget to remove it after the update is released.